### PR TITLE
Update win_run_app.py to handle llamacpp_path update

### PR DESCRIPTION
**Pull request by cherrerajobs**
Makes it work again. Sets it to empty. Maybe should be a "models" folder?

### CODE REVIEW

File: win_run_app.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ def main():
     from generate import entrypoint_main as main_h2ogpt
     os.environ['h2ogpt_block_gradio_exit'] = 'False'
     os.environ['h2ogpt_score_model'] = ''
+    os.environ['llamacpp_path'] = ''
```
Comment (pseudotensor) [inline win_run_app.py:43]: Hi, what failed for you?  Shouldn't need this.  Also, if related to recent code changes, the env would be upper-case LLAMACPP_PATH, or would be h2ogpt_llamacpp_path if trying to set those.

Comment (cherrerajobs) [reply win_run_app.py:43]: Was getting the following error:

```
Traceback (most recent call last):
  File "D:\llm\h2ogpt\win_run_app.py", line 56, in <module>
    main()
  File "D:\llm\h2ogpt\win_run_app.py", line 43, in main
    main_h2ogpt()
  File "D:\llm\h2ogpt\generate.py", line 12, in entrypoint_main
    H2O_Fire(main)
  File "D:\llm\h2ogpt\src\utils.py", line 65, in H2O_Fire
    fire.Fire(component=component, command=args)
  File "D:\miniconda3\envs\h2ogpt\lib\site-packages\fire\core.py", line 141, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "D:\miniconda3\envs\h2ogpt\lib\site-packages\fire\core.py", line 475, in _Fire
    component, remaining_args = _CallAndUpdateTrace(
  File "D:\miniconda3\envs\h2ogpt\lib\site-packages\fire\core.py", line 691, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
  File "D:\llm\h2ogpt\src\gen.py", line 1837, in main
    go_gradio(**locals())
  File "D:\llm\h2ogpt\src\gradio_runner.py", line 271, in go_gradio
    model_options0.extend(get_on_disk_models(llamacpp_path=kwargs['llamacpp_path'],
  File "D:\llm\h2ogpt\src\gen.py", line 4987, in get_on_disk_models
    llamacpp_files = [os.path.join(llamacpp_path, f) for f in os.listdir(llamacpp_path) if os.path.isfile(os.path.join(llamacpp_path, f))]
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'llamacpp_path'
```

Comment (pseudotensor) [reply win_run_app.py:43]: Ah, I just forgot to make the dir if it didn't exist, thanks

**Comment by pseudotensor** [comment]
Should be fixed: 71663f707d28a950933e195e18316286eed97f46

thanks for pointing it out!

**Pull request closed without merging**